config/boot.rb

```diff
@@ -3,3 +3,10 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../Gemfile', __dir__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+env = ENV['RAILS_ENV'] || 'development'
+if env == 'development' || env == 'test'
+    require 'bootsnap/setup'
+else
+  STDOUT.puts 'Skipping bootsnap load for non-dev environment'
```

Comment:
I would not output to STDOUT in non-dev environs, as I'm not sure of the downstream impacts in prod, CI, etc.

Reply:
Sounds good, I'll remove this line. We probably don't need it anyway, since we're only building the Gem for test/dev, it will immediately fail if we accidentally require bootsnap here on staging/prod environments